packages/vscode-extension/src/project/metro.ts

```diff
@@ -292,6 +292,82 @@ export class Metro implements Disposable {
     ws.close();
   }
 
+  private async evaluateJsFromCdp(
```

Comment:
I wonder if this is some generic method to call JS through CDP, shouldn't we put it in some helper instead of  as part of metro?

Reply:
that was my initial approach but unfortunately we need our uniq metro port that is not passed through the metro endpoints.